doc.go

```diff
@@ -43,7 +43,10 @@ It is not feasible or desirable for APIs like blob.Bucket to encompass the full
 functionality of every provider. Rather, we intend to provide a subset of the
 most commonly used functionality. There will be cases where a developer wants
 to access provider-specific functionality, such as unexposed APIs or data
-fields, errors or options.
+fields, errors or options. This can be accomplished using As functions.
+
+
```

Comment:
Nit: is this an extra CR?

Reply:
I'm following a convention we've established in other large comment blocks, see the one on top of pubsub/pubsub.go for example.

Strictly interpreting the docstring guidelines we probably don't need 2 leading newlines, but that should be a separate change. WDYT?